www/_posts/2020/Announcement for registeringpyladies-members.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post.html
+title: "Announcement for Registering PyLadies Members"
+tags: [PyLadies, Global, Organizing]
+author: Abigail Dogbe
+author_link: https://twitter.com/MesrenyameDogbe
+---
+
+Announcement for Registering PyLadies Members
+=====================
+
+This has been a busy past many months for PyLadies! 
+
+As we shared on our blog in [August 2019]( https://www.pyladies.com/blog/Important-PyLadies-Update/important-pyladies-update/), PyLadies  has been hard at work, revamping our PyLadies presence as well as our global leadership. Why? To enable us to better run our chapters ,   not only locally but globally. 
+The intent is to bring together all of our local chapters, to help us collaborate, to share ideas and also start a [PyLadies GlobalCouncil]( https://www.pyladies.com/blog/Help-us-decide-our-PyLadies-Projects-and-PyLadies-Global-Council-Selection-Process/help-us-with-pyladies-project-and-council-selection/) , which will be responsible for overseeing the Global PyLadies Community.
+
+To do this, however, we need your help! We want to engage with our PyLadies members!  Starting today, PyLadies members can register to become a basic member or a voting member.
+ 
+As  of today we have registration available in:
+
+- [English](https://forms.gle/f3M4JUzA7JH48Swo8)
+- [Spanish / Español](https://forms.gle/CaDhPsjLgEmrqV7RA)
+- Portuguese / Português](https://forms.gle/9AdTdBr67ikiAFXSA) 
+
+We’re currently in the process of translating our registration documentation into Spanish, Portuguese, Mandarin (Chinese), Arabic, and French. If you’d like to help with translation, let us know by adding your voice to the GitHub issue [here](https://github.com/pyladies/global-organizing/issues/54).
+
+
```

Comment:
Add paragraph to thank translators: 
- Portuguese: Débora Azevedo
- Spanish: Diana Rodriguez
- Russian: Volodymyr Kirichinets
- Chinese: Georgi Ker
- French: Georgi Ker

And thanks to the interim global team :)